lib/common/models/TransactionModel.ts

```diff
@@ -6,4 +6,5 @@ export default interface TransactionModel {
   transactionTime: number;
   transactionTimeHash: string;
   anchorString: string;
+  feePaid: number;
```

Comment:
We also want to add the 'normalizedFee' for this block into this model. The thinking being is that once the TransactionModel is returned to the core, it has all the data it needs to validate the feePaid vs. the normalizedFee (see #322 for details). Any idea on how much change is required for that?

Reply:
Sure, makes sense and it would not be a lot of code. This suggests we can remove the getFee() api change, right?

Reply:
Added a number, normalizedFee.